app/src/main/java/com/example/plantsapp/di/module/RepositoryModule.kt

```diff
@@ -15,12 +18,27 @@ import dagger.hilt.components.SingletonComponent
 @InstallIn(SingletonComponent::class)
 interface RepositoryModule {
 
+    @FirebaseQualifier
     @Binds
-    fun bindPlantsRepository(repo: RoomPlantsRepository): PlantsRepository
+    fun bindFirebasePlantsRepository(repo: FirebasePlantsRepository): PlantsRepository
 
+    @FirebaseQualifier
     @Binds
-    fun bindTasksRepository(repo: RoomTasksRepository): TasksRepository
+    fun bindFirebaseTasksRepository(repo: FirebaseTasksRepository): TasksRepository
 
+    @FirebaseQualifier
     @Binds
-    fun bindTasksHistoryRepository(repo: RoomTasksHistoryRepository): TasksHistoryRepository
+    fun bindFirebaseTasksHistoryRepository(repo: FirebaseTaskHistoryRepository): TasksHistoryRepository
+
+    @RoomQualifier
+    @Binds
```

Comment:
Try  @[Binds RoomQualifier]
in one line

If It works - then change other places